### PR TITLE
Separate standing color profiles per league (optional)

### PR DESCRIPTION
**Pull request by mmendelson222**
I would like to be able to distinguish between American and National League standings.  This enhancement clones the "standings" node in the colors configuration file.  In the default file, all of the colors are the same except the dividers, which are blue for NL and red for AL. 

BTW I love your change with "clinched" and "eliminated."  Do you know what support there is for post-season?

### CODE REVIEW

**Comment by WardBrian** [comment]
I haven't looked into post season yet, I think a 'bracket' wouldn't quite fit, even if you did only one league at a time

```
WC 2 -----|
          |--- Winner ---|
WC 1 -----|              | --- Winner ---|
               1 seed ---|               |
                                         | NL Champ
               2 seed ---|               |
                         | --- Winner ---|
               3 seed ---|
```

You might be able to compact this enough so its only 5 lines tall, but it would be hard.

I'll take a look at this later. Right off the bat though, I think "nl" and "al" should just be groups under standings, rather than whole separate groups

**Comment by pjockey** [comment]
Can you scroll the brackets going up or down? or section them and have them displayed on a time frame?

Also, for next year, it would be nice to add the Wild Card standings.

**Comment by WardBrian** [comment]
I don't think WCGB would fit on the standings for the division, but we could have a separate "AL Wild Card" """division""" for example which just shows the 5 teams in the wild card race. Would be interesting to try to implement

**Comment by pjockey** [comment]
> I don't think WCGB would fit on the standings for the division, but we could have a separate "AL Wild Card" """division""" for example which just shows the 5 teams in the wild card race. Would be interesting to try to implement

I played around with different variations of "divisions" and "wild card" and didn't get any joy. I'm sure I wasn't getting the proper name to place into the division in the config.json file.

**Comment by WardBrian** [comment]
There’s no such name yet, I could make one

**Comment by WardBrian** [comment]
It turns out it is also rather difficult to even get postseason data out of `statsapi` - I'll see after tonight's game if it updates at all. The wildcard standings should be pretty easy though, and I think will pair greatly with this change.

**Comment by mmendelson222** [comment]
OK great - I probably won't get to implementing your suggestion ("nl" and "al" should just be groups under standings) until tomorrow.  

re: postseason, probably `statsapi` is the big wild card.

**Comment by mmendelson222** [comment]
I went ahead and made that change - it was straightforward.  In the color config, I just added the changed color (since the nl and al amount to "overrides").

File: renderers/standings.py [hunk 1]
```diff
@@ -9,30 +9,39 @@
...
+    # try the league-specific color node.  If not present, go with the standard "standings"
+    try:
+        return colors.graphics_color(f"standings.{league}.{node_name}")
+    except KeyError:
+        return colors.graphics_color(f"standings.{node_name}")
+
+
 def render_standings(canvas, layout: Layout, colors: Color, standings: Standings, stat):
-    __fill_bg(canvas, layout, colors)
+    league = standings.preferred_divisions[standings.current_division_index].split()[0].lower()
```
Comment (WardBrian) [inline renderers/standings.py:21]:
```suggestion
    league = standings.current_standings()[:2].lower() # al or nl
```

**Review by WardBrian — CHANGES_REQUESTED** [review]
Just one simple change to use some of the existing code

**Comment by pjockey** [comment]
Screen is up and running and showing the Runs, Hits and Errors. Looks fantastic!

**Comment by WardBrian** [comment]
"NL Wild Card" is also now a valid standings option. Not super useful this season obviously

**Pull request merged**